app/src/repositories/user/userRepository.js

```diff
@@ -2,6 +2,7 @@
 
 const pool = require("@config/db");
 const toCamelCase = require("@utils/toCamelCase.js");
+const { param } = require("../../routes/home");
```

Comment:
```suggestion
const { param } = require("@routes/home");
```
경로를 alias 기반으로 변경 가능합니다.

Reply:
확인해보니까 param을 사용하지 않아서 해당 import는 삭제할 것 같습니다 ㅎㅎ...